equinix/resource_network_device.go

```diff
@@ -570,6 +618,69 @@ func createNetworkDeviceSchema() map[string]*schema.Schema {
 				},
 			},
 		},
+		networkDeviceSchemaNames["ClusterDetails"]: {
+			Type:        schema.TypeList,
+			Optional:    true,
+			ForceNew:    true,
```

Comment:
nitpick: I don't recall if ForceNew here will allow a clustername change without destroying the device first. We can introduce this new parameter with ForceNew and remove that flag later if it is a problem.

Reply:
There are no "update" API  methods for cluster details, so `ForceNew` is in fact appropriate here.